Backpack/Calendar/Classes/BPKCalendarSelectionConfigurationMultiple.m

```diff
@@ -21,7 +21,13 @@
 @implementation BPKCalendarSelectionConfigurationMultiple
 
 - (instancetype)initWithSelectionHint:(NSString *)selectionHint deselectionHint:(NSString *)deselectionHint {
-    self = [super initWithSelectionStyle:BPKCalendarSelectionStyleMultiple];
+    return [self initWithSelectionHint:selectionHint deselectionHint:deselectionHint andWholeMonthTitle:nil];
+}
+
+- (instancetype)initWithSelectionHint:(NSString *)selectionHint
```

Comment:
All of these initializers deal with accessibility and not 'visual content' 
Could we find a different way to pass in the wholeMonth title? (Maybe in the init of the calendar?)
--> This applies to all other configuration changes made in this PR

Reply:
I think this worth a longer discussion, maybe a quick zoom call?

Reply:
Discussed offline, no need to make any change.